Websocket/socket.ts

```
@@ -1,17 +1,948 @@
+                const responseEvent = {
+                  type: "workflow_json",
+                  id: workflowId,
+                  data: result,
+                  timestamp: new Date().toISOString(),
+                };
+
+                if (listeners["message"]) {
+                  const rawEvent = {
+                    toString: () => JSON.stringify(responseEvent),
+                  };
+                  listeners["message"].forEach((cb) => cb(rawEvent));
```

Biome (2.3.14)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)